### PR TITLE
Damienbod/2-2-TokenCache small improvements to readme and small code clean up

### PR DESCRIPTION
**Pull request by damienbod**
## Purpose

2-2-TokenCache small improvements to readme and small code clean up

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Comment by damienbod** [comment]
@jmprieur Is the README-incremental-instructions.md still required? Should I remove it?

Greetings Damien

File: 2-WebApp-graph-user/2-2-TokenCache/README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ It leverages the ASP.NET Core OpenID Connect middleware and Microsoft Authentica
 
 To run this sample, you'll need:
 
-- [Visual Studio 2017](https://aka.ms/vsdownload) or just the [.NET Core SDK](https://www.microsoft.com/net/learn/get-started)
+- [Visual Studio 2019](https://aka.ms/vsdownload) or just the [.NET Core SDK](https://www.microsoft.com/net/learn/get-started)
```
Comment (jmprieur) [inline 2-WebApp-graph-user/2-2-TokenCache/README.md:29]: I wonder if we want to keep giving version :)

File: 2-WebApp-graph-user/2-2-TokenCache/Startup.cs [hunk 2]
```diff
@@ -52,10 +45,10 @@ public void ConfigureServices(IServiceCollection services)
              */
 
             services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
-                    .AddMicrosoftIdentityWebApp(Configuration.GetSection("AzureAd"))
-                        .EnableTokenAcquisitionToCallDownstreamApi(initialScopes)
-                            .AddMicrosoftGraph(Configuration.GetSection("DownstreamApi"))
-                            .AddDistributedTokenCaches();
+                .AddMicrosoftIdentityWebApp(Configuration.GetSection("AzureAd"))
```
Comment (jmprieur) [inline 2-WebApp-graph-user/2-2-TokenCache/Startup.cs:48]: Actually the indentation is really based on the builders ...

**Review by jmprieur — APPROVED** [review]
LGTM. @damienbod 
Do you have a strong feeling about the indentation in the startup.cs. The builder do a find of funnel.

File: 2-WebApp-graph-user/2-2-TokenCache/README.md (hunk 1 above)
Comment (damienbod) [reply 2-WebApp-graph-user/2-2-TokenCache/README.md:29]: no I don't think this makes sense, I'll remove it

Comment (damienbod) [reply 2-WebApp-graph-user/2-2-TokenCache/README.md]: fixed

File: 2-WebApp-graph-user/2-2-TokenCache/Startup.cs (hunk 2 above)
Comment (damienbod) [reply 2-WebApp-graph-user/2-2-TokenCache/Startup.cs:48]: reverted

**Review by jmprieur — APPROVED** [review]
Thanks @damienbod for reacting so quickly to the feedback!

**Pull request merged**